### DISCUSSION TITLE
Octopus Energy HA integration on pause until April

### DISCUSSION THREAD

**Discussion by pacemaker82** [springfall2008/batpred#789] [General]
Just wanted to share https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/discussions/774

I know predbat relies on this integration and various entities - there's been some breaking changes recently from octopus which Dave quickly fixed, but looks like he's gonna take a break. Thought it good to give headsup

**Comment by iainfogg** [comment]
Good spot @pacemaker82 I've just posted on there too.

**Comment by gcoan** [comment]
Closing as historical issue
